src/main/java/seedu/saveit/model/SaveIt.java

```diff
@@ -176,6 +189,25 @@ public boolean refactorTag(Tag oldTag, Tag newTag) {
         return isEdit;
     }
 
+    /**
+     * remove the {@code oldTag} of {@code SaveIt} for all issue entries.
+     */
+    public boolean refactorTag(Tag tag) {
+        boolean isEdit = false;
+        requireNonNull(tag);
+        for (Issue issueToUpdate : issues) {
+            Set<Tag> tagsToUpdate = new HashSet<>(issueToUpdate.getTags());
+            if (tagsToUpdate.contains(tag)) {
+                tagsToUpdate.remove(tag);
+                isEdit = true;
+                Issue updateIssue = new Issue(issueToUpdate.getStatement(), issueToUpdate.getDescription(),
```

Comment:
Since both refactorTag and addTag needs to build a new issue and update issue, maybe you can consider extract them in one method